module_3/eventEmitter.js

```diff
@@ -0,0 +1,107 @@
+class EventEmitter {
+    listeners = {};  // key-value pair
+
+    addListener(eventName, fn) {
+        this.listeners[eventName] = this.listeners[eventName] || [];
```

Comment:
if value can be `null` or `undefined` I suggest use `??`:
`this.listeners[eventName] = this.listeners[eventName] ?? []`